src/vcGIS.cpp

```diff
@@ -94,3 +94,67 @@ udDouble3 vcGIS_GetWorldLocalUp(const vcGISSpace &space, udDouble3 localCoords)
 
   return udNormalize(upVector - localCoords);
 }
+
+udDouble3 vcGIS_GetWorldLocalNorth(const vcGISSpace &space, udDouble3 localCoords)
+{
+  if (!space.isProjected || space.zone.projection >= udGZPT_TransverseMercator)
+    return udDouble3::create(0, 1, 0);
+
+  udDouble3 northDirection = udDouble3::create(0, 1, 0); //TODO: Fix this
+  udDouble3 up = vcGIS_GetWorldLocalUp(space, localCoords);
+
+  udDouble3 currentLatLong = udGeoZone_CartesianToLatLong(space.zone, localCoords);
+  currentLatLong.x = udClamp(currentLatLong.x, -90.0, 89.9);
+  northDirection = udNormalize(udGeoZone_LatLongToCartesian(space.zone, udDouble3::create(currentLatLong.x + 0.1, currentLatLong.y, currentLatLong.z)) - localCoords);
```

Comment:
This is fine, but for the sake of whoever has to read this code (particularly math-heavy code), should we adopt a more verbose way of writing? For example
```
  slightlyNorthLatLong = udDouble3::create(currentLatLong.x + 0.1, currentLatLong.y, currentLatLong.z);
  slightlyNorthCartesian = udGeoZone_LatLongToCartesian(slightlyNorthLatLong);
  northVector = udNormalize(slightlyNorthCartesian - localCoords);
```